L1Trigger/Phase2L1ParticleFlow/python/l1ParticleFlow_cff.py

```diff
@@ -0,0 +1,316 @@
+import FWCore.ParameterSet.Config as cms
+
+from L1Trigger.Phase2L1ParticleFlow.pfTracksFromL1Tracks_cfi import pfTracksFromL1Tracks
+from L1Trigger.Phase2L1ParticleFlow.pfClustersFromL1EGClusters_cfi import pfClustersFromL1EGClusters
+from L1Trigger.Phase2L1ParticleFlow.pfClustersFromCombinedCalo_cfi import pfClustersFromCombinedCalo
+from L1Trigger.Phase2L1ParticleFlow.l1pfProducer_cfi import l1pfProducer
+
+# Using phase2_hgcalV10 to customize the config for all 106X samples, since there's no other modifier for it
+from Configuration.Eras.Modifier_phase2_hgcalV10_cff import phase2_hgcalV10
+from Configuration.Eras.Modifier_phase2_hgcalV11_cff import phase2_hgcalV11
+
+# Calorimeter part: ecal + hcal + hf only
+pfClustersFromCombinedCaloHCal = pfClustersFromCombinedCalo.clone(
+    hcalHGCTowers = [], hcalDigis = [],
+    hcalDigisBarrel = True, hcalDigisHF = False,
+    hadCorrector = cms.string("L1Trigger/Phase2L1ParticleFlow/data/hadcorr_barrel.root"),
+    resol = cms.PSet(
+            etaBins = cms.vdouble( 0.700,  1.200,  1.600),
```

Comment:
in principle they could be different for different instances. The code that produces these parametrizations produces simultaneously the eta bins and values (in a consistent way), and so I think it's safer to have the values rather than risk that one changes the bin edges for one of these inputs and only updates the values because the bin boundaries have been replaced by labels.